tesseract_kinematics/include/tesseract_kinematics/core/inverse_kinematics.h

```diff
@@ -151,6 +157,13 @@ class InverseKinematics
 
   /** @brief Clone the forward kinematics object */
   virtual std::shared_ptr<InverseKinematics> clone() const = 0;
+
+  /**
+   * @brief Given a solution calculate its redundant solutions
+   * @param sol The solution to calculate redundant solutions about
+   * @return A vector of redundant solutions
+   */
+  IKSolutions getRedundantSolutions(const Eigen::Ref<const Eigen::VectorXd>& sol) const;
```

Comment:
I assume this function just returns the redundant solutions, not including the original solution. If this is the case, it would good to make note of this behavior

Reply:
I did not look but is this the case for the utility function you added?

Reply:
I believe so. I made this comment before I realized this function just wraps that function

Reply:
It does not include the original solution. I have updated the documentation.